ADR/0006-log-conventions.md

```diff
@@ -150,15 +149,14 @@ func f() {
 }
 ```
 
-* By using structured logs that are only partially formatted as JSON, we should strike a balance
-  between easy readability and support for centralized queries. It should be easy for a human to
-  read pod logs of the controller directly since the first portion of each line contains a human
-  readable string early on, while the JSON formatted suffix supports the creation of queries that
-  span subsystems in centralized aggregated logs.
+* By using structured logs that are formatted entirely as JSON, we lose "easy readability" of
```

Comment:
Is the loss of readability a big deal? There's many tools that people can pipe JSON formatted logs to if needed.

Reply:
No, it's not a big deal.